.github/pull_request_template.md

```diff
@@ -2,6 +2,9 @@
 
 _Describe why you're making this change._
 
+- [ ] Updated the [CHANGELOG](https://github.com/hashicorp/terraform-provider-tfe#updating-the-changelog)
```

Comment:
```suggestion
- [ ] Update the [CHANGELOG](https://github.com/hashicorp/terraform-provider-tfe#updating-the-changelog)
```

Reply:
Yeah you can argue this one both ways. I used past tense because it's an indicator _from_ the author _to_ the reviewer that "I have updated the changelog"

If you wanted to continue with the imperative style, then it should be something like:

```
Remember to:
* Update the changelog
* Update the docs
```

Reply:
That sounds great - I like the `Remember to:` part too!

Reply:
Done